utils/cliutils/commandsflags.go

```diff
@@ -1403,6 +1412,18 @@ var flagsMap = map[string]cli.Flag{
 		Name:  Status,
 		Usage: "[Default: false] Set to true to show the status of the transfer-files command currently in progress.` `",
 	},
+	InstallPluginVersion: cli.StringFlag{
+		Name:  version,
+		Usage: "[Optional] The plugin version to download and install.` `",
```

Comment:
```suggestion
		Usage: "[Optional, default: latest] The data-transfer plugin version to download and install.` `",
```

Reply:
done the default part, since I have implemented this as a general install plugin, so I don't think we should add a specific plugin name to the usage of this general flag